test/e2e_node/jenkins/e2e-node-jenkins.sh

```diff
@@ -28,10 +28,9 @@ set -x
 
 . $1
 
-if [ "$INSTALL_GODEP" = true ] ; then
-  go get -u github.com/tools/godep
-  go get -u github.com/onsi/ginkgo/ginkgo
-  go get -u github.com/onsi/gomega
+if [ "$BUILD_GINKGO" = true ] ; then
```

Comment:
You should probably change the documentation for e2e_node test to reflect the flag change from INSTALL_GODEP.
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/e2e-node-tests.md